js/packages/teams-ai/src/Authentication.ts

```diff
@@ -151,15 +159,37 @@ export class Authentication<TState extends TurnState = DefaultTurnState> {
     /**
      * @private
      */
-    private async runDialog(context: TurnContext, state: TState, dialogStateProperty: string): Promise<DialogTurnResult<OAuthPromptResult>> {
+    private async runDialog(context: TurnContext, state: TState, dialogStateProperty: string): Promise<DialogTurnResult<any>> {
```

Comment:
`Promise<DialogTurnResult<any>>`
Is it possible to make this a union type, not `any`?